admin-dev/themes/new-theme/js/app/utils/serp/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+import Vue from 'vue';
+import serp from './serp';
+
+/**
+ * Vue component displaying a search page result, Google style.
+ * Requires a tag with the id "#serp-app" to be present in the DOM to run it.
+ * 
+ * The component is automatically updated by watching several inputs.
+ * Set the proper class to link a input to a part of the panel.
+ * 
+ * @returns {serpUtil.indexAnonym$0}
+ */
+class SerpApp {
+  constructor() {
+    // Stop if ID not found
+    if (0 === $("#serp-app").length) {
```

Comment:
I'm curious about it: as this file is only loaded in product page, in which case do you need to disable it?

Also, I'll prefer a better comment like: `Disable application If <serp /> markup is not available`

Reply:
It's written in the doc before the class

Reply:
And later, this component will be added in the SEO & URLs, then in the category page as soon as they are migrated.